scout/adapter/mongo/hgnc.py

```diff
@@ -265,9 +265,11 @@ def gene_by_alias(self, symbol, build='37'):
         Returns:
             res(pymongo.Cursor(dict))
         """
-        res = self.hgnc_collection.find({'hgnc_symbol': symbol, 'build':build})
+        LOG.debug("Fetch gene by symbol if possible: {}".format(symbol))
+        res = self.hgnc_collection.find({'hgnc_symbol': symbol, 'build': build})
         if res.count() == 0:
```

Comment:
Coud you change the code here so you are not using res.count ? You could use the sum function as in the tests